packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -866,6 +866,39 @@ class _AnimatedPaddingState extends AnimatedWidgetBaseState<AnimatedPadding> {
 /// it also requires more development overhead as you have to manually manage
 /// the lifecycle of the underlying [AnimationController].
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold}
+///
+/// The following code implements the [AnimatedAlign] as seen in the video above.
```

Comment:
They do not look the same, I will either not mention the video or make them look similar.